tests/test_fiod.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+set -xeo pipefail
+
+source tests/common.sh
+
+function finish {
+  echo "Cleaning up Fio"
+  kubectl delete -f tests/test_crs/valid_fiod.yaml
+  delete_operator
+}
+
+trap finish EXIT
+
+function functional_test_fio {
+  apply_operator
+  kubectl apply -f tests/test_crs/valid_fiod.yaml
+  check_pods 1
```

Comment:
should be 3 as it'll be 2 server pods and 1 client pods

Reply:
I will set to 2, good catch.